<?xml version="1.0" encoding="UTF-8"?><testrun name="TestGeometriaIO_Test_Version.test_GUI" project="BT-Library-Java" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="test_GUI" classname="Bt.Tests.Core.Geometry.Shapes.TestGeometriaIO_Test_Version" time="12.705">
    <error>java.util.InputMismatchException&#13;
	at java.util.Scanner.throwFor(Unknown Source)&#13;
	at java.util.Scanner.next(Unknown Source)&#13;
	at java.util.Scanner.nextInt(Unknown Source)&#13;
	at java.util.Scanner.nextInt(Unknown Source)&#13;
	at Bt.Tests.Core.Geometry.Shapes.TestGeometriaIO_Test_Version.test_GUI(TestGeometriaIO_Test_Version.java:273)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
  </testcase>
</testrun>
